content/docs/get-started/index.md

```diff
@@ -165,3 +162,363 @@ requirements:
 ```
 
 </details>
+
+<details>
+
+### Click to learn about model codification
+
+#### Why do it the MLEM way ?
+
+Saving models to files or loading them back into python objects may seem like a
+deceptively simple task at first. For example, `pickle` and `torch` libraries
+can serialize/deserialize model objects to/from files. However, MLEM adds some
+"special sauce" by inspecting the objects and serializing their metadata into
+`.mlem` files and intelligently using this later on. This metadata is necessary
+to reliably enable actions like packaging and serving of different models types
+down in various ways. MLEM allows us to automate a lot of the pain points we
+would hit later on in our ML workflow by codifying and managing this metadata
+about our models (or other objects) for us.
+
+</details>
+
+## Model Prediction
+
+Once we saved the model with MLEM we can load it to either use in our Python
+code or from the command line to generate predictions for any dataset. This
+allows us to easily decouple model training code from testing and deployment
+code.
+
+Let's try it out:
+
+<toggle>
+<tab title="Python Script">
+
+### Python code usage
+
+Load the model we saved earlier in a simple Python script to predict some
+probabilities.
+
+Create this `predict.py` script:
+
+```py
+from mlem.api import load
+
+model = load("models/rf")  # RandomForestClassifier
+features = [
+    "sepal length (cm)",
+    "sepal width (cm)",
+    "petal length (cm)",
+    "petal width (cm)",
+]
+x = pd.DataFrame([[0, 1, 2, 3]], columns=features)
+y_pred = model.predict_proba(x)
+
+print(y_pred)
+```
+
+Now, let's run the script:
+
+```cli
+$ python predict.py
+[[0.47 0.24 0.29]]
+```
+
+We see that the prediction probabilities were successfully printed to stdout.
+
+</tab>
+
+<tab title="Command Line">
+
+### Batch scoring
```

Comment:
- [ ] Not seeing a batch example..?

Reply:
this is indeed batch scoring (using the model to infer on a concrete dataset). Maybe the confusing is because the simplified example actually uses a "batch" of a single vector. the csv (or whatever format) file here can be of arbitrary size.
@aguschin  - Maybe the example should be slightly modified to emphasize it's not meant for single data-point inferencing (just `echo`ing another line should do it :D

Reply:
@jorgeorpinel will it fix your confusion? Or maybe you refered to something like `--batch X` param that will process data in chunks of len X?

```
$ mlem apply models/rf new_data.csv \
    --method predict_proba \
    --import \
    --import-type "pandas[csv]"
    --batch_size 1
```

Reply:
> the example uses a "batch" of a single vector, but the csv... can be of arbitrary size
> `    --batch_size 1`

So can we make it more realistic so that the batch is actually a batch (a group of things) ? 🙂 

But I'm getting the feeling that that's the _batchness_ aspect is not important here, which agains brings me to suggest we rename this to something like **Model scoring** (Idk, need your input to find a goof title).

Reply:
It's still very much a toy example (that's fine for the get-started imo) - but took a stab at it [here](https://github.com/iterative/mlem.ai/pull/281/files#diff-11964efe5044224d657e573702de6ffc6cdddc0b5493808fcd207bcab1b677fcR196) - simple making the dataset not a single vector but 2 should get the idea across imo

Reply:
OK from this I get that it is important to stress the batchness. Having 2 lines instead of 1 sounds simple and good. Thanks